### ISSUE TITLE
USBD480 Screen hangs

### ISSUE THREAD

**Issue by moshie** [SHWotever/SimHub#1115]
**Describe the bug**
I have a GSI Formula Elite wheel which has a USBD480 Screen when I boot up simhub initially it finds the screen and shows it as connected then after a while it changes to Searching device... and the dashboard freezes.

![image](https://user-images.githubusercontent.com/3974301/177548067-75f9f661-4f00-4698-84e5-eecfd0fb9a39.png)

Looking at the logs in simhub I can see:

```
	ERROR	USBD480 error System.Exception: ystem.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Exception: USBD480 Exception0
   at SimHub.Plugins.OutputPlugins.GraphicalDash.BitmapDisplay.USBD480.Wrapper.Check(Int32 result)
   at SimHub.BitmapDisplay.USBD480.USBD480Service.ClearScreen()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at SimHub.BitmapDisplay.Core.PipeBitmapDisplayClient`1.PipeClient_ServerMessage(NamedPipeConnection`2 connection, String message)
   at SimHub.BitmapDisplay.Core.PipeServerBase.WaitForResult[T](Func`1 action)
   at SimHub.BitmapDisplay.Core.PipeServerBase.Invoke[T](String memberName)
   at SimHub.BitmapDisplay.Core.PipeBitmapDisplayServer.ClearScreen()
   at SimHub.Plugins.OutputPlugins.GraphicalDash.RemoteDisplayManagerBase`3.ClearScreen()
   at SimHub.Plugins.OutputPlugins.GraphicalDash.BitmapDisplayBase.Worker()
```

**Simhub version**
Version: 8.0.3

**To Reproduce**
Not sure what the issue is so I am unsure how I can provide an accurate reproduction.

**Expected behavior**
When I boot simhub I expect the metrics to change on the screen when in game. Also it's able to find and retain a connection to the screen.

**Log files**
[SimHub.log](https://github.com/SHWotever/SimHub/files/9054646/SimHub.log)

**Additional context**

I also left a topic on the forum here:
https://www.simhubdash.com/community-2/simhub-support/usbd480-screen-hangs-with-exception/

I also tried an older version 7.4.0 and got the same result

Hope this is enough information let me know if you need anything further.

**Comment by moshie** [comment]
Hi @SHWotever ,

Thanks a bunch! It did end up being that I had it plugged into a usb 3.0 port.
After trying it in a usb 2.0 slot it worked.

Are there any plans to perhaps fix this for USB 3.0?

**Comment by SHWotever** [comment]
Hi thanks for the feedback, unfortunately it's not a software issue. I'm not aware of the manufacturer having planned something about it
